src/SWP/Bundle/ContentBundle/Controller/ContentPushController.php

```diff
@@ -43,23 +43,28 @@ class ContentPushController extends FOSRestController
      */
     public function pushContentAction(Request $request)
     {
-        $pipeline = (new Pipeline())
-            ->pipe([$this->get('swp_bridge.transformer.json_to_package'), 'transform'])
-            ->pipe(function ($package) {
-                $this->get('swp.repository.package')->add($package);
-
-                return $package;
-            })
-            // TODO create content component and include it into bridge bundle
-            ->pipe([$this->get('swp_content.transformer.package_to_article'), 'transform'])
-            ->pipe(function ($article) {
-                $this->get('swp.repository.article')->add($article);
-                $this->get('event_dispatcher')->dispatch(ArticleEvents::POST_CREATE, new ArticleEvent($article));
-
-                return $article;
-            });
-
-        $pipeline->process($request->getContent());
+        $content = $request->getContent();
```

Comment:
Personally i prefer Pipline solution - it's more readable and organized. We emit here events - just use them for removing already existing packages and articles.